test/ini_files/cg_any.ini

```diff
@@ -9,6 +9,8 @@ blob_bucket = elasticbeanstalk-${S3_BUCKET_ENV}-blobs
 system_bucket = elasticbeanstalk-${S3_BUCKET_ENV}-system
 metadata_bundles_bucket = elasticbeanstalk-${S3_BUCKET_ENV}-metadata-bundles
 sentry_dsn = ${SENTRY_DSN}
+identity = ${IDENTITY}
+tibanna_logs_bucket = elasticbeanstalk-${S3_BUCKET_ENV}-tibanna-logs
```

Comment:
Same issue here - this bucket is always `tibanna-logs` in the current account. We should maybe discuss how we want to handle this.

Reply:
It is perhaps not obvious, but this file only works in legacy cgap. There is a separate alpha any, and it will pick up the value of a variable that will be reliably set. This is only accommodating old code.